tasks/pkg-redhat.yml

```diff
@@ -3,7 +3,7 @@
   get_url:
     url: "{{ datadog_yum_gpgkey_new }}"
     dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
-    sha256sum: 694a2ffecff85326cc08e5f1a619937999a5913171e42f166e13ec802c812085
+    checksum: "sha256:694a2ffecff85326cc08e5f1a619937999a5913171e42f166e13ec802c812085"
```

Comment:
Could you do the same in `tasks/pkg-suse.yml` ?

Reply:
Yes I will do it today.

Reply:
Change made. Do you have a walk through how to use your tests for the this role? I'm not sure how to test the Suse change above.